benchmarks/bench_ep_matrix.py

```
@@ -0,0 +1,439 @@
+"""FlashInfer EP comm benchmark — apples-to-apples with contrib/nccl_ep/ep_bench.
+
+Measures FlashInfer's NCCL-EP dispatch+combine (comm only, NO expert FFN) the same
+way ep_bench does, so the 28-case matrix (LL em/rm @128, HT @4096/8192, ×
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)